models.py

```diff
@@ -70,6 +70,7 @@ def __unicode__(self):
 
 MARKER_TYPE_ACCIDENT = 1
 MARKER_TYPE_DISCUSSION = 2
+UNITED_HATZALA_CODE = 2
```

Comment:
I wonder why we need both this and `PROVIDER_CODE_UNITED_HATZALA` from `constants.js`.